.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.vscode/
```

Comment:
@Morrism1 check out this one https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/. You should do it in a global way